### ISSUE TITLE
Do not upgrade to 8.0.0 or above

### ISSUE THREAD

**Issue by JoshStrobl** [flathub/rest.insomnia.Insomnia#65]
@flathubbot has auto-submitted updates for 8.0.0 and 8.1.0 via #63 and #64 respectively.

Starting in 8.0.0, users will be locked out of all of their collections / requests unless they make an account (and enter payment information even if they are already part of a team). No warning is provided to the user and although they provide the option to export your content in 8.1.0's onboarding, users have been reporting content getting lost in export.

- https://github.com/Kong/insomnia/issues/6577
- https://github.com/Kong/insomnia/issues/6585
- https://github.com/Kong/insomnia/issues/6606
- https://github.com/Kong/insomnia/discussions/6590

As a result of these forced sign-in changes, a new fork called Insomnium was created by @archywillhe. This fork is more privacy respecting, dropping sentry and segment analytics, restores usage without an account, and the individual has committed to continued feature work.

- https://github.com/ArchGPT/insomnium
- https://news.ycombinator.com/item?id=37708582

Will Flathub opt to migrate users over to Insomnium, keep this locked to < 8, or some other action? Unless users of Insomnia follow Hacker News, they likely will be unaware of any of these major changes and even if correctly going through the export process, could result in data loss for the user, greatly impacting their ability to do their work.

**Comment by JoshStrobl** [comment]
Thoughts @barthalion, I noticed you are a recent contributor that has push capabilities.

**Comment by JoshStrobl** [comment]
Wow hey, thanks for the quick response, it's much appreciated. It looks like Insomnium generates appimages so I'll see about putting something together since this flatpak is using that already. I use Insomnia for work so I sound like a good candidate for maintainership of the not-bork version :D

**Comment by Eonfge** [comment]
For reference, Flatpak and Flathub has a maintainer and migration policy in place:
- A maintainer can step forward and package [Insomnium](https://github.com/ArchGPT/insomnium)
- If no maintainer steps up for Insomnia, this package can be archives
- Existing users can be migrated to Insomnium

I've done something similar with Xournal and Xournal++. I helped in maintaining Xournal++ and when nobody cared about maintaining the legacy version, we moved everybody over and archives the old version.

@jovial-snyder is right though: As long as this package is called Insomnia, it should package upstream's product with all anti-featues and enshittification included. Only when properly forking and adding a new trademark, can Flathub consider a migration.
